server/src/main/java/org/apache/iotdb/db/service/DataNode.java

```diff
@@ -286,6 +286,11 @@ private void setUp() throws StartupException, QueryProcessException {
 
     registerManager.register(ReceiverService.getInstance());
 
+    // in cluster mode, RPC service is not enabled.
+    if (IoTDBDescriptor.getInstance().getConfig().isEnableRpcService()) {
+      registerManager.register(RPCService.getInstance());
+    }
+
```

Comment:
`RPCService` has already registered in `setUpRPCService()` method.

Reply:
OK. I check this.

Reply:
I have  just removed this part of codes.